Library/Homebrew/commands.rb

```diff
@@ -203,7 +203,7 @@ def command_description(command, short: false)
 
     if (cmd_parser = Homebrew::CLI::Parser.from_cmd_path(path))
       if short
-        cmd_parser.description.split(".").first
+        cmd_parser.description.split(/\.(?>\s|$)/).first
```

Comment:
@ZhongRuoyu Could this get a Ruby comment added above clarifying what this is doing and why? It's not obvious from reading the code alone. Thanks!